rest/summaries.go

```diff
@@ -0,0 +1,26 @@
+package polygon
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/polygon-io/client-go/rest/client"
+	"github.com/polygon-io/client-go/rest/models"
+)
+
+const (
+	GetSummariesPath = "/v1/summaries"
+)
+
+// SummariesClient defines a REST client for the Polygon Snapshot Summary API.
+type SummariesClient struct {
+	client.Client
+}
+
+// GetSummaries retrieves summaries for the ticker list with the given params
```

Comment:
Nit: add a period at the end of this sentence (it'll render to `... given params For more details ...` in the godocs)